src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -178,6 +184,15 @@
     <Compile Include="$(CommonPath)\Interop\Unix\libssl\Interop.SafeSslHandle.cs">
       <Link>Interop\Unix\libssl\Interop.SafeSslHandle.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libssl\SecuritySafeHandles.cs">
+      <Link>Interop\Unix\libssl\SecuritySafeHandles.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libssl\StreamSizes.cs">
+      <Link>Interop\Unix\libssl\StreamSizes.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libssl\SslConnectionInfo.cs">
+      <Link>Interop\Unix\libssl\SslConnectionInfo.cs</Link>
```

Comment:
For the original Win32 implementation these are structs extracted from schannel.h and so they must be in the Windows\Interop. The requirement is to rename them to the original form in the .h include (not done in this PR).

This is only phase 1 of the refactoring required for the Nego work for Schannel. We can further refine this afterwards. My recommendation is to remove these from the common code (possibly replacing them with a PAL equivalent).